src/applications/disability-benefits/disability-rating-calculator/tests/00-all-fields.e2e.spec.js

```diff
@@ -0,0 +1,64 @@
+const E2eHelpers = require('platform/testing/e2e/helpers');
+const Timeouts = require('platform/testing/e2e/timeouts.js');
+// const DrcHelpers = require('../utils/helpers.js');
+// const ENVIRONMENTS = require('site/constants/environments');
+// const FormsTestHelpers = require('platform/testing/e2e/form-helpers');
+const inputs = require('./utils/inputs.json');
+const DrcE2eHelpers = require('./drc-helpers.js');
+
+module.exports = E2eHelpers.createE2eTest(client => {
+  // Navigate to beta-page.
+  client
+    .openUrl(`${E2eHelpers.baseUrl}/disability/about-disability-ratings-beta/`)
+    .waitForElementVisible('body', Timeouts.normal)
+    .assert.title('How VA Assigns Disability Ratings | Veterans Affairs');
+
+  // Test row deletion.
+  // TODO: Cover edge-case* for deleting AFTER calculating combined rating.
+  // *Design Team to decide what should happen for this edge-case.
+  client
+    .waitForElementPresent('.btn-add', Timeouts.slow)
+    .expect.element('.btn-add').to.be.visible;
+  DrcE2eHelpers.fillRatings(client, { ratings: [30, 20, 10] });
```

Comment:
Can we move the `ratings` here into a top-level constant, along with the resultant combined disability rating? Similar to how we do it in `src/applications/disability-benefits/disability-rating-calculator/tests/utils/inputs.json`.

Reply:
Replaced w/ const ref.